spec/integration/assert_called_spec.rb

```diff
@@ -38,9 +39,8 @@
 
     describe 'assertion message' do
       it 'provides a helpful message' do
-        expect(first_command.inspect).to eql '<Stubbed "first_command">'
```

Comment:
We can move this stripped down version of the StubbedCommand into the matcher if you think it takes away too much information (you can override @actual there)